### PR TITLE
Fix #20349: Tile inspector creates map animations

### LINKED ISSUE #20349: Tile inspector does not create map animations when copying animated elements
### Operating System

Any

### OpenRCT2 build

OpenRCT2, v0.4.5 (66ae20b47 on develop, DEBUG)

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

This bug is a graphical glitch or error

### Describe the issue

See the attached video:

[anim.webm](https://github.com/OpenRCT2/OpenRCT2/assets/7877312/c68de92f-56de-4ad5-bf6b-43ded4537bef)

### Steps to reproduce

1. Copy an animated element with the tile inspector
2. Paste an animated element with the tile inspector

### Attachments

[anim.webm](https://github.com/OpenRCT2/OpenRCT2/assets/7877312/c68de92f-56de-4ad5-bf6b-43ded4537bef)

### PR DESCRIPTION
**Pull request by mrmbernardi**
This PR fixes the issue #20349

### CODE REVIEW

File: src/openrct2/world/MapAnimation.h [hunk 1]
```diff
@@ -43,3 +45,4 @@ void MapAnimationCreate(int32_t type, const CoordsXYZ& loc);
 void MapAnimationInvalidateAll();
 const std::vector<MapAnimation>& GetMapAnimations();
 void MapAnimationAutoCreate();
+void MapAnimationAutoCreateTileElement(TileCoordsXY coords, TileElement* el);
```
Comment (duncanspumpkin) [inline src/openrct2/world/MapAnimation.h:48]: I feel the name is slightly odd it suggests its creating a tile element. Maybe stick a `For` in there. `MapAnimationAutoCreateForTileElement`

**Review by duncanspumpkin — APPROVED** [review]
Seems sensible. Increment the network version as well.

File: src/openrct2/world/MapAnimation.h (hunk 1 above)
Comment (mrmbernardi) [reply src/openrct2/world/MapAnimation.h]: I've just changed the name to `MapAnimationAutoCreateAtTileElement`

**Comment by mrmbernardi** [comment]
I've bumped `NETWORK_STREAM_VERSION` to 7

**Comment by ZehMatt** [comment]
Thanks

**Pull request merged**